goreleaserlib/goreleaser.go

```diff
@@ -26,7 +26,7 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-var pipes = []pipeline.Pipe{
+var pipes = []pipeline.Piper{
 	defaults.Pipe{},  // load default configs
```

Comment:
Maybe it would be nicer if `defaults` is not a `Piper` and instead of explicitly listing

Reply:
the problem is the order of things, the default's order is different from the pipe's order...

Reply:
Ah missed that part. Never mind then :)